rules/windows/builtin/win_susp_local_anon_logon_created.yml

```diff
@@ -18,7 +18,7 @@ logsource:
 detection:
     selection:
         EventID: 4720
-        SAMAccountName: '*ANONYMOUS*LOGON*'
+        SAMAccountName|contains: 'ANONYMOUS*LOGON'
```

Comment:
same here. as you are using `contains`, it stops to process masks.
I would reconstruct it this way:

```
        SAMAccountName|contains|all: 
            - 'ANONYMOUS'
            - 'LOGON'
```

Reply:
Fixed :)